### PR TITLE
SingleSpaceAfterConstructFixer - Add support for `namespace`

### LINKED ISSUE #3728: Multiple spaces after namespace keyword
I noticed that PHP-CS-Fixer does not fix multiple spaces after the `namespace` keyword.

#### The PHP version you are using (`$ php -v`):

> PHP 7.0.30 (cli) (built: Apr 26 2018 11:49:41) ( NTS )

#### PHP CS Fixer version you are using (`$ php-cs-fixer -V`):

> PHP CS Fixer 2.11.1 Grey Devil by Fabien Potencier and Dariusz Ruminski

#### The command you use to run PHP CS Fixer:

> vendor/friendsofphp/php-cs-fixer/php-cs-fixer fix

#### The configuration file you are using, if any:

Using ruleset `@PSR1`, `@PSR2` and `@Symfony`, otherwise pretty standard.

#### If applicable, please provide minimum samples of PHP code (as plain text, not screenshots):

 * before running PHP CS Fixer (no changes):
```php
namespace  My\Name\Space;
```

 * with unexpected changes applied when running PHP CS Fixer:
```php
namespace  My\Name\Space;
```

 * with the changes you expected instead:
```php
namespace My\Name\Space;
```

### PR DESCRIPTION
**Pull request by julienfalque**
Closes #3728.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Isn't changing the default a breaking change?

File: tests/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixerTest.php [hunk 1]
```diff
@@ -3183,6 +3183,66 @@ public function provideCommentsCases()
...
+            '<?php
+namespace Foo {}
+namespace Bar {}',
+            '<?php
+namespace
+Foo {}
+namespace
+Bar {}',
+        ];
+    }
```
Comment (GrahamCampbell) [inline tests/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixerTest.php:3244]: Missing some cases:

```php
<?php

namespace // comment
Foo;
```

```php
<?php

namespace /* comment */ Foo;
```

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: tests/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixerTest.php (hunk 1 above)
Comment (julienfalque) [reply tests/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixerTest.php:3257]: Added, thank :+1:

**Comment by julienfalque** [comment]
> Isn't changing the default a breaking change?

This already happened (e.g. #5406) but I would say it's fine here: it's part of the initial intent of this rule but was overlooked. So this PR is a an enhancement/bugfix rather than a new feature.

**Review by kubawerlos — APPROVED** [review]

**Comment by keradus** [comment]
Thank you @julienfalque.

**Pull request merged**